### PR TITLE
docs: :memo: edits to tidymodels section

### PR DESCRIPTION
**Pull request by AndersAskeland**
## Description

Some rephring and corrections.

This PR needs a quick eview.

## Checklist

- [x] Ran spell-check
- [x] Formatted Markdown
- [x] Rendered website locally

### CODE REVIEW

**Comment by AndersAskeland** [comment]
Can't build website locally right now due to some error in `r3admin::read_common()` . @lwjohnst86 do you know what going on?

File: sessions/stats-analyses-basic.qmd [hunk 1]
```diff
@@ -60,9 +60,9 @@ Specific "anti"-objectives on things that we will *not* learn:
     tests.
 -   How to interpret the statistical results.
 
-This means that what we show is *for demonstration purposes only*, they
-could be entirely wrong in how to do them correctly if an expert were to
-review them.
+This means that what we show is *for demonstration purposes only*. Our
+implementation and chooice of model and analyses could be wrong if an
```
Comment (lwjohnst86) [inline sessions/stats-analyses-basic.qmd:64]:
```suggestion
implementation and choice of model and analyses could be wrong if an
```

File: sessions/stats-analyses-basic.qmd [hunk 2]
```diff
@@ -449,16 +446,16 @@ lipidomics
...
+minutes to think and respond.
 :::
 
-The first observation isn't always an issue and depends heavily on the
-model type you use. Since we are using logistic regression, the model
-assumes that each row is an individual person. But our data is in the
-long format, so each person has multiple rows. The second observation is
-that there seems to be a data input error, since there are multiple
-`Cholesterol` values, while all other metabolites only have one:
+The first issue, isn't always an issue and depends heavily on the model
```
Comment (lwjohnst86) [inline sessions/stats-analyses-basic.qmd:453]:
```suggestion
The first issue, which isn't always an issue, depends heavily on the model
```

**Comment by lwjohnst86** [comment]
Yea, you need to install dependencies again since I updated r3admin recently

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
:tada: nice! Only small edits

**Comment by AndersAskeland** [comment]
> Yea, you need to install dependencies again since I updated r3admin recently

Nice, working now :)

**Pull request merged**